dpgen/generator/arginfo.py

```diff
@@ -101,8 +101,33 @@ def training_args() -> List[Argument]:
 
 
 # Exploration
+def model_devi_jobs_template_args() -> Argument:
+    doc_template = 'Give a template for 01.model_devi.'
+    doc_template_lmp = 'The path to input.lammps template'
+    doc_template_plm = 'The path to input.plumed template'
+
+    args = [
+        Argument("lmp", str, optional=True, doc=doc_template_lmp),
+        Argument("plm", str, optional=True, doc=doc_template_plm),
+    ]
+    return Argument("template", list, args, [], optional=True, repeat=False, doc=doc_template)
+
+
+def model_devi_jobs_rev_mat_args() -> Argument:
+    doc_rev_mat = '' # looking forward to update
+    doc_rev_mat_lmp = '' # looking forward to update
+    doc_rev_mat_plm = '' # looking forward to update
+
+    args = [
+        Argument("lmp", dict, optional=True, doc=doc_rev_mat_lmp),
+        Argument("plm", dict, optional=True, doc=doc_rev_mat_plm),
+    ]
+    return Argument("rev_mat", list, args, [], optional=True, repeat=False, doc=doc_rev_mat)
+
+
 def model_devi_jobs_args() -> List[Argument]:
     # this may be not correct
+    doc_sys_rev_mat = "" # looking forward to update
```

Comment:
doc_sys_rev_mat = 'system-resolved revise matrix for revising variable(s) defined in the template into specific values. Values should be individually assigned to each system adopted by this iteration, through a dictionary where first-level keys are values of `sys_idx` of this iteration.